GA/HeaderFiles/DataWrite.h

```diff
@@ -25,78 +29,75 @@ void DataWrite(std::vector<std::vector<std::vector<float> > >& varVector, std::v
   generationDNA << "Working on behalf of Dr. Amy Connolly" << "\n";
   generationDNA << "And the ANITA project" << "\n";
   generationDNA << "Revision date: 21 March 2018 1800 EST" << "\n";
-	
-  /*for(int i=0;i<freq_coeffs;i++)
-    {
-      if(i==freq_coeffs-1)
-	{
-	  generationDNA << freqVector[i] << "\n";
-	}
-      else
-	{
-	  generationDNA << freqVector[i] << ",";
-	}
-     }*/
-     
-  
   generationDNA << "Frequencies go here:\n";
   generationDNA << "Matrices for this Generation:\n";
-  for(int i=0;i<population;i++)
+
+  // Write DNA
+  for (int i = 0; i < population; i++)
+  {
+    for (int j = 0; j < sections; j++)
     {
-      for(int j=0;j<sections;j++)
-	{
-	  for(int k=0;k<genes;k++)
-	    {
-	      if(k==(genes-1))
-		{
-		  generationDNA << varVector[i][j][k] << "\n";
-		}
-	      else
-		{
-		  generationDNA << varVector[i][j][k] << ",";
-		}
-	    }
-	}
+      for (int k = 0; k < genes; k++)
+      {
+        if (k == (genes - 1))
+        {
+          generationDNA << dna_output[i][j][k] << "\n";
+        }
+        else
+        {
+          generationDNA << dna_output[i][j][k] << ",";
+        }
+      }
     }
+  }
   generationDNA.close();
-  
-  
-  // PARENT LOCATION FILE
- 
+
+
+  // Write parent file
+  ofstream Parents;
+  Parents.open("parents.csv");
+  Parents << "Location of individuals used to make this generation:" << endl;
```

Comment:
Did you want to add the location before the `endl` here?

Reply:
This is correct. The csv is containing those locations relative to the child that was produced from those individuals. I could probably clean up the wording and format thought if it is confusing.